src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styles from '@/components/Divider/Divider.module.scss'
+
+interface DividerProps {
+  mode?: 'horizontal' | 'vertical'
+}
+
+const Divider: React.FC<DividerProps> = ({ mode = 'horizontal' }) => {
```

Comment:
We don't have a Divider component in the Makaira Admin UI but nice to have.

The mode doesn't seam to have any effect. Please add the styling for vertical mode or remove the mode prop.